docs/enterprise/about.md

```diff
@@ -0,0 +1,53 @@
+---
+title: About
+sidebarDepth: 1
+description: What does the Pomerium Enterpise Console offer?
+---
+
+# Pomerium Enterprise
+
+This paragraph introduces what Pomerium Enterprise is.
+
+## Features
+
+In addition to the capabilities provided by open-source Pomerium, Pomerium Enterprise provides:
+
+### Management GUI
+
+The Pomerium Enterprise Console lets you view traffic and logs, define routes and policies, and organize your service access all from an intuitive web interface.
+
+![Overview animation of the Pomerium Enterprise Console](./img/console-overview.gif)
+
+### Programmatic API
+
+<!-- @Travis please add details. -->
+
+### Session management
+
+Quickly view who is logged in your infrastructure, with easy access to revoke sessions.
+
+![Pomerium Enterprise Console Session List](./img/console-session-list.png)
+
+### Self-Service, and Access Controls
```

Comment:
See background for more details: https://www.notion.so/pomerium/Permissions-Access-Control-e32ff518f1564b3698d13611c449b436